crowbar_framework/Gemfile

```diff
@@ -35,6 +35,8 @@ gem "simple-navigation", "~> 3.12"
 gem "simple_navigation_renderers", "~> 1.0"
 gem "sqlite3", "~> 1.3"
 gem "syslogger", "~> 1.6"
+gem "yaml_db", "~> 0.3.0"
```

Comment:
Looks like that gaiting is complaining about this: "`resolve': Could not find gem 'yaml_db (~> 0.3.0) ruby'`" that is part of